tools/upgrade/migration/resource/1.4.0/04_label_updates.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+oc label $(oc get all -l 'syndesis.io/app=syndesis' -o name) --overwrite=true app=syndesis
+oc label persistentvolumeclaim/syndesis-db --overwrite=true syndesis.io/type=infrastructure
```

Comment:
Why is the syndesis.io/app=syndesis label missing on this PVC ? And if so, shouldn't we add it as well ?

Reply:
I did a diff of the [current template](https://github.com/syndesisio/syndesis/blob/71664f1448dbb0a0e9ab2221e7b7259c6bd5abff/install/syndesis.yml) and the [1.3.x template](https://github.com/syndesisio/syndesis/blob/4c5b88e8e6ce9987b8c0233d4868d59febc305b5/install/syndesis.yml). It seems that every object that was annotated with `syndesis.io/app=syndesis` also got a `app=syndesis`, we added `syndesis.io/type=infrastructure` to the PVC. Why was something missing or why it was added is a lengthier discussion, probably we were trying to be a bit uniform with labelling.